test/integration/features/user_adds_template_to_project.spec.js

```diff
@@ -42,24 +38,42 @@ describe('User adds a template to a project', function() {
             // verify we have the 2 deployments in the template
             var deploymentsPage = new DeploymentsPage(project);
             deploymentsPage.visit();
-            expect(element(by.cssContainingText('td', 'mongodb')).isPresent()).toBe(true);
-            expect(element(by.cssContainingText('td', 'nodejs-mongodb-example')).isPresent()).toBe(true);
+            expect(element(by.cssContainingText('td', 'mongodb')).isPresent()).toBe(true); // TODO: use fixture
```

Comment:
That comment means to eventually use the fixture (json) and pluck out the names, rather than hard-coding `mongodb`, `nodejs-mongodb-example`. 

Is on my list to add some table helpers as well.